rfcs/0175-appeals-council.md

```diff
@@ -0,0 +1,170 @@
+
+1) We, as moderators, commit to basing decisions on general principles and concrete reasons that apply equally to all community members, irrespective of characteristics such as sociopolitical orientation, economic status, identity, or religion.
+2) We adhere to the principles of *equality* and *objectivity*. Equality refers to the status of each community member, while objectivity pertains to the consistent, unbiased reasoning behind our decisions.
+3) A *concrete reason* is a verifiable state of affairs that can be demonstrated to other community members upon request.
+4) We acknowledge that *concrete reasons* are not self-explanatory, except in extreme cases, and we will provide explanations relating them to the general principles guiding our decisions.
+5) We affirm that moderation actions must be fair and proportionate to the related issue.
+6) We ensure that concrete reasons for permanent bans are available for inspection by long-standing community members.
+7) We recognize that moderation actions should not be based solely on the volume of complaints.
+8) We commit to treating community members with respect and avoiding derogatory language when addressing concerns.
+9) We affirm our role as moderators is to facilitate a respectful community, not to punish members. We will use language that reflects mutual respect.
+10) We prefer inclusion to exclusion, resorting to permanent exclusion only in cases of extreme necessity. We are committed to exploring ways to prevent exclusion and to engaging constructively in efforts to reinstate excluded members.
+11) To reflect a consensus in a diverse community, the moderation team must embody cultural and viewpoint diversity, respecting members with varying societal views, provided they do not undermine equal community membership.
+12) We respect viewpoint diversity in cultural and political matters when selecting moderation team candidates.
+13) We understand that reasonable moderation is based on past events or future expectations, necessitating respect for historical records and transparency in controversial events.
+14) We uphold that true diversity necessitates exceptional tolerance, actively embracing not only the ideas we favor but, also, those we disagree with. This commitment to tolerance is fundamental in fostering a community where diverse viewpoints can coexist and enrich our collective experience.
+
+## Resolution to Adopt the *Memorandum*
+
+- Incorporate the *Memorandum* text into the moderation repository on GitHub, linking it prominently from the Code of Conduct and all relevant public information sources.
+- Implement one-year term limits for moderation team members, with a one-year hiatus before potential return. Apply this rule retroactively.
```

Comment:
So you want to just eject all the mods and make it so they cant come back for at least a year?

Reply:
I think that rotation is a good thing. Provided the moderation team is always accountable for their actions, this would minimize both arbitrariness and incentives for power tripping.

Reply:
> I think that rotation is a good thing. Provided the moderation team is always accountable for their actions, this would minimize both arbitrariness and incentives for power tripping.

Rotation only works if there is people that would do the Job. Moderation in NixOS spaces is volunteering work and so far the moderation team struggles a lot with member retention and acquisition and moderators semi-quickly burning out. This also is a limiting factor in terms of how much work can be put into reports, if every single mute/suspension/ban has to be a 5 pages report then you will only burn out people more quickly.

Reply:
Well moderation is a responsibility, a job of sorts after all. It requires commitment and collecting/presenting evidence that justifies your actions, particularly if these actions would prevent interaction with the codebase permanently, is part of that commitment. You cannot act on a whim and expect people to just trust you. I think accountability mechanisms should be in place.

Reply:
Given that currently moderators rarely stay for longer than half a year before burning out, this rule is mostly void and I don't really see much point in discussing it.

Reply:
Then why not include it?  If moderators wouldn't mind it, and it would improve trust in moderators.

Reply:
> Given that currently moderators rarely stay for longer than half a year before burning out

Maybe this is a symptom of how behaviors are enforced in the community. Not reprimanding and establishing behavioral precedents early on opens up the pandora's box of "what is acceptable".

Unfortunately, I think the environment has shifted from "what is acceptable" to "who his acceptable".